### PR TITLE
New: Add navi-admin routes

### PR DESCRIPTION
**Pull request by ldkge**
Resolves #

## Description
Adds navi-admin admin and roles routes

## Proposed Changes

- add admin route to host admin panel
- add roles route to host roles dashboard

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/admin/tests/dummy/mirage/serializers/application.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import BaseJSONSerializer from 'navi-core/mirage/serializers/base-json-serializer';
+
+export default BaseJSONSerializer;
```
Comment (kevinhinterlong) [inline packages/admin/tests/dummy/mirage/serializers/application.js:3]:
```suggestion
export default class extends BaseJSONSerializer {}
```

File: packages/admin/addon/routes/admin.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class Admin extends Route {
```
Comment (kevinhinterlong) [inline packages/admin/addon/routes/admin.ts:9]:
```suggestion
export default class AdminRoute extends Route {
```

File: packages/admin/addon/router.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/**
```
Comment (kevinhinterlong) [inline packages/admin/addon/router.js:1]: Use typescript for this file

File: packages/admin/addon/templates/admin.hbs [hunk 4]
```diff
@@ -0,0 +1 @@
+{{outlet}}
```
Comment (mattgraham1995) [inline packages/admin/addon/templates/admin.hbs:1]: ©️

File: packages/admin/addon/templates/admin/roles.hbs [hunk 5]
```diff
@@ -0,0 +1 @@
+{{outlet}}
```
Comment (mattgraham1995) [inline packages/admin/addon/templates/admin/roles.hbs:1]: ©️

**Pull request merged**